src/components/forms/DateInput/DateInput.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { DateInput } from './DateInput'
+import { DateInputGroup } from '../DateInputGroup/DateInputGroup'
+import { Fieldset } from '../Fieldset/Fieldset'
+
+export default {
+  title: 'Form/DateInput',
```

Comment:
I just noticed because this is `Form` and not `Forms` this is resulting in a separate Storybook folder. not a blocker but we should clean it up